docs/tutorials/fundamentals/viewer.md

```diff
@@ -232,42 +215,34 @@ viewer.add_points(points, size=30)
 :tags: [hide-input]
 nbscreenshot(viewer, alt_text="points layer showing 3 white points layered on top of astronaut image in napari viewer")
 ```
+   
++++  
 
-+++
-
-Adjusting these properties in the layers list will cause corresponding changes
-to properties on the selected individual layers. These properties are also able
-to be changed and accessed in the console through `viewer.layers`.
+Adjusting these properties in the layers list will cause corresponding changes to properties on the selected individual layers. These properties can also be changed and accessed in the console through `viewer.layers`.  
 
-For example, the name and opacity of a layer can be changed within the console
-as follows:
+For example, the name and opacity of a layer can be changed within the console as follows:
 
 +++
 
-```python
+```{code-cell} python
 viewer.layers[0].name = 'astronaut'
 viewer.layers[0].opacity = 0.7
 ```
 
 ```{code-cell} python
 :tags: [remove-cell]
-
 viewer.close()
 ```
 
-and these changes will instantly propagate to the GUI. For more information
-about the different properties for different layer types please see our
-[guides on using layers](../../howtos/layers/index).
+and these changes will instantly propagate to the GUI. For more information about the different properties for different layer types please see our layer specific tutorials listed at the bottom of this tutorial. 
 
 +++
 
-### Layer buttons
+### Create Layer buttons
 
-New `Points`, `Shapes`, and `Labels` layers can be added to the viewer using the
-layer buttons between the layer controls and layer list. These correspond to
-the following calls:
+New `Points`, `Shapes`, and `Labels` layers can be added to the viewer using the layer buttons between the layer controls and layer list. These correspond to the following calls:
 
-```python
+```{code-cell} python
```

Comment:
The {code-cell} statement is what makes the green highlight appear on the left side.  It works in other docs, I can't see why it's not working here.

Reply:
It doesn't work because this code is not supposed to be run - it's an incomplete example. Whenever you use "python" here, this means "print this text verbatim, using python syntax highlighting". If you use "code-cell" it means "run this code as a standalone program and show me any outputs". This is why this cell is generating an error below, and subsequently causing other cells not to execute properly.

Reply:
Ok - here's the update:

The above commands all work on a napari console, but not directly from an ipython console. Possible solutions:

1. Change the text to read
````
New empty `Points`, `Shapes`, and `Labels` layers can be added to the viewer using the layer
buttons between the layer controls and layer list. This is equivalent to, for example, the following
code in the napari console to make an empty `Points` layer:

```python
viewer.add_points()
```

```{note}
The above commands will only work on a napari console. If you are running a script or typing these
commands on an ipython console, you will need to provide a `data` argument to the
`viewer.add_points` command. See the :func:`napari.viewer.add_points` documentation for more
details.
```

Once added, either in the GUI or via the console, these layers become accessible in the layers list
section of the GUI and at `viewer.layers`.
````

2. Delete this information entirely. I think this is misleading - how many people will bump into this exact behavior and not understand that you can have different requirements depending on which console you are calling these commands from?
3. Do 1., but expand this text to explain WHY this happens

Personally I'd lean into 1, and do 3 on a follow-up. What do you think @psobolewskiPhD?

Reply:
@melissawm It's just `add_labels` that requires `data`. 
Here's copy paste from ipython:
```
In [2]: import napari

In [3]: viewer = napari.Viewer()

In [4]: viewer.add_points()
Out[4]: <Points layer 'Points' at 0x1068904c0>

In [5]: viewer.add_shapes()
Out[5]: <Shapes layer 'Shapes' at 0x179c847f0>
```
For me, all of the above works in ipython and in the napari-console in 0.4.17 and on main.

So the Note isn't needed.

In general, I'm not sure how likely anyone is to programmatically call those, but I think the point here is just to show that everything the GUI does is doing something that one could do code. So it's more about the concept.

BTW, on that note, maybe make that last sentance:
```
Once added, either in the GUI or via the console, these layers become accessible in the layers list section of the GUI and in `viewer.layers`. 
For example, the empty Points layer created above can be accessed using `viewer.layers['Points']`.
```

Reply:
If this is just an example, maybe the code block should do away with the boiler plate?
Just be:
```
viewer.add_points()
```